### ISSUE TITLE
Foreign Key Constraints Enforced or Not? SQLite with SQL Tools

### ISSUE THREAD

**Issue by mrslleigh** [mtxr/vscode-sqltools#1384]
I have tested SQL Tools a little as a possible alternative to the, unmaintained, extension I am using to teach SQL to pupils in VS Code. I can't find anywhere in the settings to set the pragma settings to on but I can successfully just run the SQL statement and can run a check and get a 1 as a response suggesting that this has been successful. However, when testing to see if I can remove a parent record before removing the child record, it lets me do that which I don't think it should.

I don't know if this is an issue or a feature request I'm afraid, I just wanted to reach out in case there is any easy answer.

**Comment by zliebersbach** [comment]
Yeah I don't think it's possible to adjust the SQLite driver settings to enable foreign key constraints 😕

**Comment by zliebersbach** [comment]
Could probably be submitted as a PR, need to set the params [here](https://github.com/mtxr/vscode-sqltools/blob/4d95f465dddc644880c82f24cc9f2b29e42a8a85/packages/driver.sqlite/src/ls/driver.ts#L44)
